### ISSUE TITLE
Justice League Heroes - Option to export model with Skeleton Data/Weights?

### ISSUE THREAD

**Issue by 0Curtain0** [bigianb/bgda-explorer#12]
Option to export character models with bone weights (skeleton data)? 
This can help reduce the amount of work needed to rig character models.

**Comment by bigianb** [comment]
At least for BGDA we have this data ...  It just needs exporting. Need to figure out a suitable format.

**Comment by bigianb** [comment]
latest release has the JLH animation data parsed so you can export posed meshes. What's the most useful format for skinned models? FBX?

**Comment by BryceBarbara** [comment]
FBX or GLTF would be best.

**Comment by bigianb** [comment]
I implemented GLTF export in the Java code last year sometime ... https://github.com/bigianb/bgda-explorer/blob/master/bgtools/src/main/java/net/ijbrown/bgtools/vifview/Demo.java

**Comment by BryceBarbara** [comment]
Cool! Looks like there is a [pretty good C# library](https://github.com/vpenades/SharpGLTF) that handles the GLTF implementation details. Since GLTF supports files with multiple animations, we should start thinking of how we might want to support exporting all applicable animations in the LMP file.

**Comment by HonorsoftEntertainment** [comment]
I agree that FBX would be good format for bone-weights and skeleton data, maybe GLTF, but DAE might be the easiest (but not preferable) format to implement/
